translators/src/Print/Agda.hs

```diff
@@ -17,7 +17,7 @@ newtype Agda ann = Agda {get :: Doc ann}
 class Keywords rep where
   import_ :: rep
   assign  :: rep
-  rec     :: rep
+  recrd   :: rep
```

Comment:
For consistency with `import_` and `data_`, we should call this `record_`

Reply:
Agree - but will implement that in the Pretty1 branch instead, as otherwise this is going to cause difficult merges.